docs/docs/cmd/spo/file/file-list.md

```diff
@@ -16,6 +16,10 @@ m365 spo file list [options]
 `-f, --folder <folder>`
 : The server- or site-relative URL of the folder from which to retrieve files
 
+`-r, --recursive [recursive]`
```

Comment:
Since this is a switch we shouldn't include `[recursive]` because that would make the option expect a value like `--recursive true`

Reply:
Fixed